include/benchmark/benchmark.h

```diff
@@ -1122,7 +1124,17 @@ class Benchmark {
   callback_function setup_;
   callback_function teardown_;
 
-  Benchmark& operator=(Benchmark const&);
+  Benchmark(Benchmark const&)
+#if defined(BENCHMARK_HAS_CXX11)
+      = delete
+#endif
+      ;
+
+  Benchmark& operator=(Benchmark const&)
+#if defined(BENCHMARK_HAS_CXX11)
+      = delete
+#endif
+      ;
```

Comment:
Are these changes required here?

Reply:
Not quite sure. I stumbled upon this while fixing the DLL export under MSVC.

Reply:
For reference, this are the linker errors triggered without these changes:

```console
Creating library src\benchmark.lib and object src\benchmark.exp
string_util.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
sysinfo.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
timers.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
perf_counters.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
reporter.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
sleep.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
statistics.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
console_reporter.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
counter.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
csv_reporter.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
json_reporter.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
benchmark_runner.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
colorprint.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
commandlineflags.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
complexity.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
benchmark.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
benchmark_api_internal.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
benchmark_name.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
benchmark_register.cc.obj : error LNK2001: unresolved external symbol "protected: __cdecl benchmark::internal::Benchmark::Benchmark(class benchmark::internal::Benchmark const &)" (??0Benchmark@internal@benchmark@@IEAA@AEBV012@@Z)
src\benchmark.dll : fatal error LNK1120: 1 unresolved externals
```

Reply:
I would still like to hear an explanation as to why this is okay and not does result in ABI mismatch.

Reply:
Looks like MSVC does not allow to set the language standard prior to C++11. Therefore, an ABI mismatch cannot generally occur.